### PR TITLE
Add meta tags headers for link unfurling

### PR DESCRIPTION
**Pull request by enzofab91**
## What does this PR do?
This PR adds meta tags for better link unfurling. In this first version, title and description are customized for each page. But at the moment, a preview image can be seen only in product metrics page.

The chart image generation is done with [Quickchart](https://quickchart.io/) API. This library generate an image from a javascript chart like the ones generated in the views with Chartkick.

**Here are some examples previews:**

When sharing product dashboard:

Screen Shot 2021-09-10 at 16 45 19

When sharing defect escape rate metric detail:

Screen Shot 2021-09-10 at 16 50 30

When sharing development cycle metric detail:

Screen Shot 2021-09-10 at 17 23 40

When sharing planned to done ratio metric detail:

Screen Shot 2021-09-10 at 17 01 37

## Notes:
Create the following env variable after deploying `QUICKCHART_URL` with value https://quickchart.io/chart/create.

Resolves #[EM-216](https://rootstrap.atlassian.net/browse/EM-216): Add OG meta tags for better link unfurling.

### CODE REVIEW

File: app/helpers/meta_tags_helper.rb [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+    generate_url_image if current_page?(products_metrics_development_metrics_path)
+  end
+
+  private
+
+  def title_for_pages
+    if current_page?(repositories_development_metrics_path)
+      "#{@repository[:name]} summary"
+    elsif current_page?(open_source_index_path)
+      'Open Source repositories'
```
Comment (lauperalti) [inline app/helpers/meta_tags_helper.rb:34]: What do you think about using `I18n` for plain text occurrences such as titles and descriptions?

Comment (enzofab91) [reply app/helpers/meta_tags_helper.rb]: Done!

File: app/services/builders/chartkick/generate_chart_image.rb [hunk 2]
```diff
@@ -0,0 +1,119 @@
+module Builders
+  module Chartkick
+    class GenerateChartImage
+      QUICKCHART_URL = 'https://quickchart.io/chart/create'.freeze
```
Comment (lauperalti) [inline app/services/builders/chartkick/generate_chart_image.rb:4]: Maybe be in  `.ENV`?

**Review by lauperalti — APPROVED** [review]
Very nice job! 👏

File: app/services/builders/chartkick/generate_chart_image.rb (hunk 2 above)
Comment (enzofab91) [reply app/services/builders/chartkick/generate_chart_image.rb]: Done!

Comment (lauperalti) [reply app/services/builders/chartkick/generate_chart_image.rb]: Remember to add it to `.env.example`

**Review by sebastiancaraballo — APPROVED** [review]
Good. However, I feel `app/helpers/meta_tags_helper.rb` class should be refactored. Try calling methods dynamically instead of having `if/else` blocks. Also, analyze whether it's a good idea to split this class or not.

**Pull request merged**